src/frontend/admin/src/services/api/models/Order.ts

```diff
@@ -35,6 +35,16 @@ export type Order = AbstractOrder & {
     issued_on: string;
   };
   main_invoice: OrderMainInvoice;
+  has_consent_to_terms: boolean;
+  contract?: OrderContractDetails;
+};
+
+export type OrderContractDetails = {
+  definition_title: string;
+  id: string;
+  organization_signed_on?: string;
+  student_signed_on?: string;
+  submitted_for_signature_on?: string;
```

Comment:
Any value are optional they are nullable.

```suggestion
  has_consent_to_terms: boolean;
  contract: OrderContractDetails | null;
};

export type OrderContractDetails = {
  definition_title: string;
  id: string;
  organization_signed_on: string | null;
  student_signed_on: string | null;
  submitted_for_signature_on: string | null;
```